test/e2e/apps/replica_set.go

```diff
@@ -35,7 +36,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *extensions.ReplicaSet {
+func newRS(rsName string, replicas int32, rsPodLabels map[string]string, imageName string, image string) *apps.ReplicaSet {
```

Comment:
ok, here's where we may need to gate now... tests using apps.ReplicaSet won't run on old clusters, right?